orm/src/test/java/org/hibernate/search/test/spatial/SpatialIndexingTest.java

```diff
@@ -180,6 +182,7 @@ public void testDistanceSort() throws Exception {
 
 	@Test
 	@TestForIssue(jiraKey = "HSEARCH-1708")
+	@Category(ElasticsearchSupportInProgress.class)
```

Comment:
Probably a short comment why it's part of this category will help to remember the context when getting back to it.